apps/edxapp/templates/services/cms/macros/mongodb_host.yml.j2

```
@@ -0,0 +1,16 @@
+{# compute mongodb host #}
+{%- macro mongodb_host(env_type, trashable_env_types, deployment_stamp, endpoint_mongodb_ips, edxapp_mongodb_port, edxapp_mongodb_host) %}
+    {%- if env_type in trashable_env_types -%}
+        {{ edxapp_mongodb_host }}-{{ deployment_stamp }}:{{ edxapp_mongodb_port }}
+    {%- else -%}
+        {%- set mondogdb_service_name = "%s-%s" | format(edxapp_mongodb_host, deployment_stamp) -%}
+        {%- set ns = namespace(hosts="") -%}
+        {%- for ip in endpoint_mongodb_ips -%}
+            {%- set ns.hosts = "%s%s:%s" | format(ns.hosts, mondogdb_service_name, edxapp_mongodb_port | int + loop.index0) -%}
```

That double typo saved you :wink: 

```suggestion
            {%- set ns.hosts = "%s%s:%s" | format(ns.hosts, mongodb_service_name, edxapp_mongodb_port | int + loop.index0) -%}
```